docs/content/dagster-plus/managing-deployments/branch-deployments/using-branch-deployments-with-github.mdx

```diff
@@ -30,87 +33,108 @@ To complete the steps in this guide, you'll need:
 
 ## Step 1: Generate a Dagster+ agent token
```

Comment:
Do we still recommend creating a dedicated agent for this for Hybrid?

Reply:
I don't know the answer to this. My guess is yes, since none of the work I did recently for branch deployments affected agents, but maybe there was some other work that has happened that i'm unaware of. Maybe @gibsondan will know

Reply:
I think so yes, it will make it less likely that your branch deployments eat up all your resources for your prod cluster